rest/server_context.go

```diff
@@ -978,15 +980,19 @@ func (sc *ServerContext) startStatsLogger() {
 }
 
 func (sc *ServerContext) stopStatsLogger() {
-	if sc.statsLoggingTicker != nil {
-		sc.statsLoggingTicker.Stop()
+	if sc.statsContext.statsLoggingTicker != nil {
```

Comment:
Since we're adding a statsContext, should stopStatsLogger and the various new add...Stats() below be functions on the statsContext instead of the serverContext?

Reply:
Agreed.  Fixed.  I moved the low hanging fruit for now.   There's other code that could also be moved, but it would need to take a `ServerContext` reference (or internal fields of `ServerContext`), and it feels too much like a circular dependency.